packages/amplify-e2e-core/src/categories/api.ts

```diff
@@ -303,9 +303,30 @@ export function addRestApi(cwd: string, settings: any) {
           .sendLine('n');
       }
 
+      chain.wait('Restrict API access');
+
+      if (settings.restrictAccess) {
+        chain.sendConfirmYes().wait('Who should have access');
```

Comment:
Multiline it please, like below

Reply:
Discussed on Slack, but mentioning it here - prettier unfortunately changed this back after trying to make the change.